components/inputs/input-text.js

```diff
@@ -454,12 +459,7 @@ class InputText extends FocusMixin(LabelledMixin(FormElementMixin(SkeletonMixin(
 		super.updated(changedProperties);
 
 		changedProperties.forEach((oldVal, prop) => {
-			if (prop === 'unit') {
-				if (this.unit && this.unit !== '%') {
-					throw new Error('Invalid unit value for d2l-input-text.');
```

Comment:
We now need to relax the accepted values for `unit`. I think the reason we were worried about opening this up was that consumers would provide values that made no sense when read with the label. `unit-label` solves that.

Reply:
Updated this to now require `unit-label` unless `unit` is `%`.